utilities/install.py

```diff
@@ -87,6 +87,7 @@ def compile_mod():
 def main(netpyne_branch, workspace_branch, geppetto_branch=None, skipNpm=False,
          skipTest=False, development=False):
     cprint("Installing requirements")
+    print(workspace_branch)
```

Comment:
Should this print be here? (again, sorry if my tone sounds bad or something, I really don't know if it makes sense, I'm still new on a lot of things in NetPyNE-UI)